src/pages/components/select/usage.mdx

```diff
@@ -442,13 +442,13 @@ More information about designing for AI guidelines is coming soon.
 
 <Tab label="Default">
 
-![Enabled state of default select with AI presence](images/select-usage-AI-presence-default.png)
+![Default select with AI presence](images/select-AI-presence-default-usage.png)
```

Comment:
In the image, in the menu option 2 should be selected since that's what in the top select. Those should match. Same goes for the revert image.
<img width="760" alt="image" src="https://github.com/carbon-design-system/carbon-website/assets/11670886/564537a5-0cf0-4762-be1f-2f09a06b8e9b">

Reply:
Oh yeahh, good catch. 
Updated it, thank you!